packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -779,9 +782,12 @@ const selfBehavior = {
    * In extreme situations, system health may require liquidating all vaults.
    * This starts the liquidations all in parallel.
    *
-   * @param {MethodContext} context
    */
-  liquidateAll: async ({ state, facets: { helper } }) => {
+  async liquidateAll() {
+    const {
+      state,
+      facets: { helper },
+    } = this;
```

Comment:
@turadg, all my comments were on the individual commits. Some don't appear on the whole diff. If it would help, I can copy them over to improve visibility.

Reply:
I think they all came through. I'll address before merging.